bloom-housing/bloom
# fix: new lottery copy

## Description

There isn't a ticket for this work, we just had [copy edits](https://docs.google.com/document/d/130OuxoQnKnukoVr_SPQq57A0JZ4vycdcS9q0EX2Gkso/edit#heading=h.z2we68wtdnff) and some tiny design changes come in from Doorway on existing work.

They aren't called out explicitly in the doc, but I've listed them below.

## How Can This Be Tested/Reviewed?

* Slight string adjustments to wording and removing one date
* Removing references to Doorway / BAHFA in the strings so we don't have them in core, but adding them as custom strings we can change in Doorway
* Making the tabs lowercase
* Updating the add application modal to check the new listingLotteryStatus
* Changing the run with duplicates button back to an alert color

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
